src/JsonSchema/Constraints/Factory.php

```diff
@@ -82,6 +87,23 @@ public function __construct(
     }
 
     /**
+     * Get package files
+     *
+     * @param string File path, relative to the root of the package
+     */
+    public function getPackageFile($path)
```

Comment:
Looks like it would work, but isn't this repeating some of the work done in `SchemaStorage`?